index.js

```diff
@@ -582,7 +582,9 @@ const renderBody = (title, body, role, config, alerts, req) =>
     alerts,
     layout: body,
   });
-const wrapIt = (config, bodyAttr, headers, title, body) => `<!doctype html>
+const wrapIt = (config, bodyAttr, headers, title, body) => {
+  config["backgroundColorDark"] = "#182433";
```

Comment:
set this instead in an onDone on the configuration workflow: https://github.com/saltcorn/saltcorn/blob/60545ee8f4e2099d6f9568cc71093a2988f2a2c0/packages/server/routes/fields.js#L246

onDone = (ctx)=> {ctx.backgroundColorDark = "..."}